repairnator/src/main/java/fr/inria/spirals/repairnator/process/git/GitHelper.java

```diff
@@ -11,6 +11,7 @@
 import okhttp3.Response;
 import org.eclipse.jgit.api.Git;
 import org.eclipse.jgit.api.MergeCommand;
+import org.eclipse.jgit.api.MergeResult;
```

Comment:
It was not on purpose. I was trying to do something after merge but then I gave up for now and I forgot to remove that import.